src/arch/arm/pagetable.hh

```diff
@@ -161,31 +165,19 @@ struct V8PageTableOps64k : public PageTableOps
     LookupLevel lastLevel() const override;
 };
 
-// ITB/DTB table entry
-struct TlbEntry : public Serializable
+struct TlbEntry;
+
+class TlbTypes
```

Comment:
It has to be noted that we still use TlbEntry though... What shall we do about it?

Reply:
That one too. Basically, my request was to change any Tlb to TLB, to standardize the naming scheme. I understand if you would rather leave TlbEntry for the future, as it is a class that already existed before this PR, so it may generate some merge conflicts.

Reply:
Amended the TlbTypes. Will update TlbEntry with a separate PR